module/x/gravity/keeper/ethereum_event_handler.go

```diff
@@ -44,6 +45,19 @@ func (k Keeper) Handle(ctx sdk.Context, eve types.EthereumEvent) (err error) {
 		if err := k.bankKeeper.SendCoinsFromModuleToAccount(ctx, types.ModuleName, addr, coins); err != nil {
 			return err
 		}
+
+		ctx.EventManager().EmitEvent(sdk.NewEvent(
```

Comment:
Is is possible to implement in the hook, so we don't need to patch gravity-bridge?

Reply:
Yes I think so. But not sure if we want to patch the other places with weird unit64 to int cast as well.

Alternatively I can implement the event emit on Cronos hook first; then create a PR to the upstream and we can patch them in next iteration. What do you think?

Reply:
Open this PR on Cronos side: https://github.com/crypto-org-chain/cronos/pull/133

See which way we want to go